Basic/2DSpaceShooter/ProjectSettings/ProjectSettings.asset

```diff
@@ -959,7 +959,7 @@ PlayerSettings:
   qnxGraphicConfPath: 
   apiCompatibilityLevel: 6
   captureStartupLogs: {}
-  activeInputHandler: 0
+  activeInputHandler: 2
```

Comment:
Is there a reason why this is set to "both"? (IIRC there was a UIToolkit bug that didn't allow stuff [textboxes?] to work if input was set to "new only", is it because of that?)

![image](https://github.com/user-attachments/assets/a669c03a-6e65-4b13-baeb-77bd0518d72a)

Reply:
Thanks for pointing that out, I overlooked that. I had this error:
Yeah I have mouse input issues if I switch to only the new one only 
![image](https://github.com/user-attachments/assets/bbdf5736-04d5-47eb-b192-a6392447be31)
 But it is fixed now that I adjusted the EventSystem within the scene to use the new input system

Reply:
works perfectly now!